python/py-pymatgen/Portfile

```diff
@@ -50,10 +50,6 @@ if {${name} ne ${subport}} {
         port:py${python.version}-sympy \
         port:py${python.version}-tabulate \
         port:py${python.version}-uncertainties
-
-    if {${python.version} < 38} {
-        depends_run-append port:py${python.version}-typing_extensions
-    }
-
-    livecheck.type  none
+ 
+   livecheck.type  none
```

Comment:
this indentation doesn't seem right, but also it's nowadays set automatically by the `python` PG so you can remove the `livecheck.type` line

Reply:
I'll remove the line. See 9adf22d.